cilium/lb/main.go

```diff
@@ -123,7 +136,7 @@ func init() {
 						Usage: "Delete all entries",
 					},
 				},
-				ArgsUsage: "--all | <address> <port> <slave-index>",
+				ArgsUsage: "--all | <address>:<port>",
```

Comment:
ah I see. I think we should keep it as `<address> <port>` (like `ncat`) to avoid confusion whether the user should have entered the `[` and `]` after he realized that `ff00::1:8080` doesn't work. This way it doesn't matter if it's is IPv4 or IPv6. It's my opinion, if you like that way it's fine as well.